### ISSUE TITLE
Prevent that every number followed by a dot is interpreted as order list

### ISSUE THREAD

**Issue by rico** [markedjs/marked#924]
Is there a way to prevent marked from rendering / interpreting every number followed by a dot as an ordered list item?

The problem is, that here in Switzerland we format a date as follows: **14. August 2017**.  Now if I run this through marked I get **1. August 2017** (first item of ordered list). Not exactly what I want :-)

**Comment by joshbruce** [comment]
See #982

**Comment by bambooom** [comment]
Hi, I'm still seeing this issue, [demo on marked.js.org](https://marked.js.org/demo/?text=%23%20Some%20title%0A%0AThe%20development%20work%20of%20the%20something%20something%20%20started%20in%20%0A2016.%20It%20was%20pioneered%20by%20somebody%20and%20some%20others.%0AOver%20the%20past%202%20years%20or%20so%2C%20a%20lot%20of%20bugs%20have%20been%20fixed%20in%0Amode%0Aand%20our%20recent%20extensive%20testings%20show%20that%20it%20is%20already%20mature%20enough%0Afor%20our%20own%20production%20use.%0A&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3Afalse%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20true%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=master)

What's my input:

```
# Some title

The development work of the something something  started in 
2016. It was pioneered by somebody and some others.
Over the past 2 years or so, a lot of bugs have been fixed in
mode
and our recent extensive testings show that it is already mature enough
for our own production use.
```

What I get:
![Screenshot 2019-04-26 at 1 27 56 PM](https://user-images.githubusercontent.com/14840170/56785301-21b47a80-6827-11e9-9944-c0b5194aa5f7.png)

However, `2016.` is just the end to the previous sentence, it should not be rendered as ordered list. I do can just move it to the previous line for now, but how should I prevent this later? Thanks.

**Comment by UziTech** [comment]
looks like this is fixed when `gfm: false` [demo](https://marked.js.org/demo/?text=%23%20Some%20title%0A%0AThe%20development%20work%20of%20the%20something%20something%20%20started%20in%20%0A2016.%20It%20was%20pioneered%20by%20somebody%20and%20some%20others.%0AOver%20the%20past%202%20years%20or%20so%2C%20a%20lot%20of%20bugs%20have%20been%20fixed%20in%0Amode%0Aand%20our%20recent%20extensive%20testings%20show%20that%20it%20is%20already%20mature%20enough%0Afor%20our%20own%20production%20use.%0A&options=%7B%0A%20%22gfm%22%3A%20false%0A%7D&version=master)

**Comment by UziTech** [comment]
Looks like this was fix in [v0.7.0](https://github.com/markedjs/marked/releases/tag/v0.7.0)
